database/migrations/20190723105203-create-user.js

```diff
@@ -1,51 +1,59 @@
+/* eslint-disable arrow-body-style */
+/* eslint-disable object-shorthand */
+/* eslint-disable func-names */
 /* eslint-disable no-unused-vars */
+
```

Comment:
Why are these rules disabled?

Reply:
I was having some lint errors so I had to disable it, let me see if i can refactor it a bit more

Reply:
@mazma1 I have refactored it and removed the eslint comments